Source/SA/DPD/DPD.csproj

```
@@ -0,0 +1,77 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{C725B272-463E-421D-8648-4E3F87868711}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>DPD</RootNamespace>
+    <AssemblyName>DPD</AssemblyName>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <FileAlignment>512</FileAlignment>
+    <Deterministic>true</Deterministic>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
+    <Optimize>false</Optimize>
+    <OutputPath>bin\Debug\</OutputPath>
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+    <OutputPath>bin\Release\</OutputPath>
```

Per [Contributing](https://github.com/NISystemsEngineering/rfic-reference-design-libraries-csharp/blob/master/CONTRIBUTING.md#building-and-preparing-for-distribution) this should be:
```suggestion
    <OutputPath>..\..\..\Builds\</OutputPath>
```

---

Hey @cindyxie, it looks like you changed this, but for the wrong build configuration (Debug instead of Release). You'll have to switch the build configuration before setting this property. For now, set it back to `\bin\Debug` for debug builds and `..\..\..\Builds\` for the Release build.

---

Thanks @mdwhitten. Please review https://github.com/NISystemsEngineering/rfic-reference-design-libraries-csharp/pull/63/commits/8ac5b207a37c90734ca7bced06abc342300fd790 and resolve this conversation if fixed.